### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/cg-landing#58]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://cloud.gov/:
- **Error:** Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
  - WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
  - html > body > header > div > h2
  - `<h2 class="header-title"> <a href="/" class="logo"...</h2>`
- **Error:** This fieldset element does not have a name available to an accessibility API. Valid names are: legend element.
  - WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Fieldset.Name
  - #contact > fieldset
  - `<fieldset> <h4>Sign up for update...</fieldset>`
- **Error:** Fieldset does not contain a legend element. All fieldsets should contain a legend element that describes a description of the field group.
  - WCAG2AA.Principle1.Guideline1_3.1_3_1.H71.NoLegend
  - #contact > fieldset
  - `<fieldset> <h4>Sign up for update...</fieldset>`
## Summary:
- 3 Errors

**Comment by msecret** [comment]
This was captured in separate issues
